config/crds/cluster.k8s.io_machinedeployments.yaml

```diff
@@ -18,7 +18,6 @@ spec:
       labelSelectorPath: .status.labelSelector
       specReplicasPath: .spec.replicas
       statusReplicasPath: .status.replicas
-    status: {}
```

Comment:
What happened to the status subresource?

Reply:
Good point, this needs to be added back. It was conflicting when both v1alpha2 and v1alpha2 were in the same CRD